compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -2435,6 +2484,11 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
   }
 
   // Virtual
+  loco::NodeShape visit(const luci::CircleBidirectionalSequenceLSTMOut *node) final
```

Comment:
please put this block after `CircleOutput `

Reply:
will address this, thanks.